CHANGES.md

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
Did you push the commit replaceing the Unreleased to 1.1.0? This changelog entry should go to the new "Unreleased" and the current "Unreleased" should be "1.1.0"